### ISSUE TITLE
Accessibility: Keyboard Only: Accordions in sidebar cannot be accessed via a keyboard

### ISSUE THREAD

**Issue by alisterscott** [Automattic/wp-calypso#1999]
This issue has been split out from: https://github.com/Automattic/wp-calypso/issues/697

Accordions in sidebar cannot be accessed via a keyboard. Also, the source order seems off.

**Comment by rachelmcr** [comment]
I was able to access the accordions in the editor sidebar with a keyboard, but only after manually confirming that the accordion section I wanted was in focus. To resolve this we could improve the `:focus` styles for the accordion sections.

#bug-scrub #pre-gutenburg-bug-purge
